packages/orbit-components/src/Box/Box.stories.tsx

```diff
@@ -250,10 +270,24 @@ export const Positions = () => {
   );
 };
 
-export const PaddingMargin = () => {
-  const margin = select("margin", Object.values(SPACINGS), SPACINGS.SMALL);
-  const padding = select("margin", Object.values(SPACINGS), SPACINGS.SMALL);
```

Comment:
This was broken in SB as it was not possible to set padding, should be fixed now.